frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -36,47 +44,40 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
     setMode(value as MODES);
   };
 
-  React.useEffect(() => {
-    k8sGet(ClusterServiceVersionModel, appName, ns)
-      .then((clusterServiceVersionObj) => {
-        // Todo(bipuladh): Remove this check in 4.7
-        const isIndependentSupported = getAnnotations(clusterServiceVersionObj)[
-          OCS_SUPPORT_ANNOTATION
-        ].includes('external');
-        if (isIndependentSupported) {
-          setIndependent(true);
-          const { configMaps = [], secrets = [], storageClasses = [] } = getRequiredKeys(
-            clusterServiceVersionObj,
-          );
-          setIndependentReqdKeys({ configMaps, secrets, storageClasses });
-          setDownloadFile(
-            createDownloadFile(
-              getAnnotations(clusterServiceVersionObj)?.[
-                'external.features.ocs.openshift.io/export-script'
-              ],
-            ),
-          );
-        }
+  const csvResource = useDeepCompareMemoize<WatchK8sResource>(
+    {
+      kind: referenceForModel(ClusterServiceVersionModel),
+      name: appName,
+      namespace: ns,
+      isList: false,
+    },
+    true,
+  );
+
```

Comment:
`useK8sWatchResource` already uses `useDeepCompareMemoize`.
```suggestion
  const csvResource = 
    {
      kind: referenceForModel(ClusterServiceVersionModel),
      name: appName,
      namespace: ns,
      isList: false,
    };

```